src/vs/platform/windows/electron-main/windowImpl.ts

```diff
@@ -1050,6 +1050,8 @@ export class CodeWindow extends Disposable implements ICodeWindow {
 		configuration.maximized = this._win.isMaximized();
 		configuration.partsSplash = this.themeMainService.getWindowSplash();
 
+		configuration.accessibilitySupport = app.isAccessibilitySupportEnabled();
```

Comment:
I'm not sure this is needed - it didn't seem to fix anything 🤔 but @deepak1556 pls lmk your thoughts

Reply:
I was thinking of removing https://github.com/microsoft/vscode/blob/224472b8b914862214db6faa9385afa40427acf8/src/vs/platform/windows/electron-main/windowImpl.ts#L1086 in favor of the current change as it will cover both load and reload path. @bpasero if we update the configuration during load path then is it necessary to define the property at the window open call site, can we remove those in favor of this change ?